### ISSUE TITLE
Add ability to add reason for false positive 

### ISSUE THREAD

**Issue by jsoref** [zaproxy/zaproxy#6737]
**Is your feature request related to a problem? Please describe.**
Right now I can mark a lot of things as false positives, but I expect someone to ask me "why is this a false positive".

**Describe the solution you'd like**
 I'd like a clear-text field, or maybe a clear text field + a link field so that I can explain why something is a false positive.

I might use a link to link to a ticket in zaproxy where i've reported the false positive.

**Describe alternatives you've considered**

**Additional context**
#3319 is potentially related, if I report something as a real issue, I'd want to be able to link to the ticket responsible for fixing it.

**Would you like to help implementing this feature?**

**Comment by thc202** [comment]
Candidate for alert tags.

**Comment by ricekot** [comment]
@jsoref Alert Tags are a new feature which will allow you to add key-value pairs to each alert. You can try the latest weekly and they will also be available starting 2.11.0. Do you think they will satisfy your needs?

**Comment by jsoref** [comment]
Yes. Can I easily apply them at the same time I mark a false positive?

(I'm currently fighting other things, so I don't really have time to test, although if someone wants to allocate an hour or so to walk through the app w/ me, i'm happy to invest that time on various things.)

**Comment by ricekot** [comment]
Yeah, you just need to double click the alert to edit it, scroll down, and add a new tag to the 'Alert Tags' table.

**Comment by jsoref** [comment]
Am I right in assuming I can't filter by tags?

e.g. Hide all Alerts w/ a give tag?

Personally, I think I'd probably call it a `label` and not a `tag`. Offhand, I can't explain _why_ I'd do that, but...

**Comment by kingthorin** [comment]
> Am I right in assuming I can't filter by tags?

Not currently but it’s something that could be looked at.

**Comment by jsoref** [comment]
With that, the "requirements" here would be addressed.
